libqtile/widget/sensors.py

```diff
@@ -81,6 +80,7 @@ def __init__(self, **config):
     def _configure(self, qtile, bar):
         self.unit = "°C" if self.metric else "°F"
         base.InLoopPollText._configure(self, qtile, bar)
+        self.foreground_normal = self.foreground
```

Comment:
Any reason why we need to make this a variable anyways? I saw only one use for foreground_normal

Reply:
It's certainly not necessary to have it but it does, potentially, make the code a little clearer as it differentiates the different foreground values.

Reply:
Yeah I was looking for something to nitpick here, whatever works for me honestly it doesn't matter much